psi4/driver/driver_cbs.py

```diff
@@ -429,15 +554,15 @@ def _get_default_xtpl(nbasis: int, xtpl_type: str) -> Callable:
         raise ValidationError(f"Stage treatment must be 'corl' or 'scf', not '{xtpl_type}'")
 
 
-def _validate_cbs_inputs(cbs_metadata, molecule):
+def _validate_cbs_inputs(cbs_metadata: CBSMetadata, molecule: Union["qcdb.Molecule", core.Molecule]) -> CBSMetadata:
```

Comment:
It's a string to avoid importing. It might be that way to avoid circular imports. I have a shelved pass to clean up imports and `import *` in the driver, and I hope to clean this up then. afaik, mypy and sphinx treat the string like the class.